src/components/ResponsiveAppBar.jsx

```diff
@@ -43,11 +44,12 @@ const ResponsiveAppBar = () => {
     setAnchorElNav(null);
   };
 
-  const { devMode: isDevMode } = React.useContext(DevModeContext);
+  const { showDatabase } = React.useContext(DevModeContext);
+  const { showNutriscore } = React.useContext(DevModeContext);
 
-  const displayedPages = pages.filter(
-    (page) => page.url !== "insights" || isDevMode
-  );
+  const displayedPages = pages
+    .filter((page) => page.url !== "insights" || showDatabase)
+    .filter((page) => page.url !== "nutriscore" || showNutriscore);
```

Comment:
I thought I merge [my PR](https://github.com/openfoodfacts/hunger-games/pull/93/files), but I did not

In the PR I propose to introduce a new parameter `devModeOnly` to the page's objects. Such that we could do 
```jsx
const displayedPages = pages.filter((page) => !page.devModeOnly || isDevMode);
```

Instead of adding a variable for each page, we create, we could use an object `visiblePages` such that `visiblePages = { nutriscore: true, insights: false}` means nutriscore page is visible and insight is not

Then your code would look like

```suggestion
  const { devMode: isDevMode, visiblePages } = React.useContext(DevModeContext);

  const displayedPages = pages
    .filter((page) => !page.devModeOnly || (isDevMode && visiblePages[page.url] ))
```

Reply:
So should I include this new devmode code here or are you merging your other pr?

Reply:
I will merge my PR after this one to avoid you git conflicts to manage 😉